deployments/container/nvidia-mig-manager-example-hopper-blackwell.yaml

```diff
@@ -294,23 +300,23 @@ data:
           mig-devices:
             "2g.47gb": 3
 
-      all-3g.95gb:
+      all-3g.93gb:
```

Comment:
Instead of updating `all-3g.95gb` to  `all-3g.93gb`, let's add `all-3g.93gb` to this file and leave the former as-is

Reply:
@tariq1890 Please see if my latest push is ok now